PeerioTechnologies/peerio-icebear
# Feat no multiple snackbars for folder download

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/13616/desktop-downloading-a-folder-with-multiple-files-should-only-show-one-snackbar

#### Testing instructions  

- create a folder
- upload multiple files there
- go to parent folder
- select a folder with checkbox
- download a folder

Only 1 snackbar (Downloads complete) should appear.


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
